android/README.md

```diff
@@ -3,7 +3,7 @@
 ## Prerequisites
 * Android Studio 4.1+
 * Java SE Development Kit 8 
-* [Node.js](https://nodejs.org/) 8.9+ (for building KeymanWeb)
+* [Node.js](https://nodejs.org/) 14.17+ (for building KeymanWeb)
```

Comment:
Is this strictly correct? @jahorton

Reply:
fwiw, I updated my development machine to node 16.20 in order to get npm 7 (discussed in #5092)

Reply:
FWIW:

<img width="322" alt="Screen Shot 2021-05-21 at 1 07 27 PM" src="https://user-images.githubusercontent.com/25213402/119090211-87d75000-ba35-11eb-992f-5f457b4338bf.png">

Reply:
Also, NPM 7 shouldn't be a hard requirement.  Its new package lockfile format is designed to be backward-compatible with NPM 5 and 6, though they won't be able to process some of its enhancements.  I put a link in #5092 about that.

Reply:
From https://nodejs.org/en/about/releases/
I'm fine keeping the current 14 LTS.

Node.js 12 ceases to be maintained Apr 2022.

While the current package lockfile is backward-compatible with NPM 5 and 6, might as well have all developers use NPM 7 so we don't have to wrangle with package lockfile diffs.